remote-access/samba.md

```diff
@@ -0,0 +1,99 @@
+## Samba/CIFS
+
+Samba is a implementation of the SMB/CIFS networking protocol which is used by Windows devices to providing shared access to files, printers, and serial ports etc. There is a comprehensive [Wikipedia page](https://en.wikipedia.org/wiki/Samba_(software)) which describes Samba and its capabilities.
+
+This page will explain how to use a subset of the Samba system to 'mount' a shared folder on a Windows device so it appears on your Raspberry Pi, or to share a folder on your Raspberry Pi so it can be accessed by a Windows Client.
+
+### Installing CIFS/Samba support
+
+By default, Raspbian does not include CIFS/Samba support, but it can easily be added. The following commmand will install all the required components for using Samba as a server or a client.
+```
```

Comment:
Best to use `bash` as the "language" of the code block for syntax highlighting purposes